src/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs

```diff
@@ -1789,7 +1789,6 @@ public static void DCS_KeyValuePair()
     }
 
     [Fact]
-    [ActiveIssue("ReflectionBasedDCS")]
     public static void DCS_ConcurrentDictionary()
```

Comment:
This particular test probably was enabled by the previous two fixes related to IDictionary.

What I did was, after each fix, we tried to remove ActiveIssue attributes from test cases as many as possible. I might missed removing the attribute from some test cases for one fix, but I removed it in the iterations after that.